src/clients/FtpGateway.cc

```diff
@@ -532,7 +532,10 @@ ftpListParseParts(const char *buf, struct Ftp::GatewayFlags flags)
     ftpListParts *p = NULL;
     char *t = NULL;
     const char *ct = NULL;
```

Comment:
If you have not already, please check whether this variable is still used.

Reply:
yes it is used only inside "/* Try EPLF format ... */" block
I hesitated for a while, but finally I decided to move its declaration inside this block.
commit 9b29918163